packages/amplication-git-utils/src/git/git-client.ts

```diff
@@ -0,0 +1,44 @@
+import { simpleGit, SimpleGit } from "simple-git";
+
+export class GitClient {
+  git: SimpleGit;
+
+  constructor() {
+    this.git = simpleGit();
+  }
+
+  async clone(cloneUrl: string, cloneDir: string): Promise<void> {
```

Comment:
don't you want to return on any phase some value or indication of the action ?

Reply:
No, because the success of this method is for itself the indication